pom.xml

```diff
@@ -166,9 +166,9 @@
         </dependency>
 
         <dependency>
-            <groupId>org.apache.commons</groupId>
-            <artifactId>commons-io</artifactId>
-            <version>1.3.2</version>
+            <groupId>commons-io</groupId>
+		    <artifactId>commons-io</artifactId>
+		    <version>2.5</version> <!-- 2.5 is the last version to support java 1.6 -->
```

Comment:
I think this might be able to be switched to a test only dependency. It could also be a runtime dependency of the HttpClient though so I didn't change it.